src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,18 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        Library library = new Library();
+
+        while (library.stop) {
+            library.add();
+        }
+
```

Comment:
Przenieś tę pętlę do środka `Library`, natomiast na zewnątrz możesz opakować to w metodę `library.run()`